docs/topics/email.txt

```diff
@@ -282,8 +282,12 @@ All parameters are optional and can be set at any time prior to calling the
   new connection is created when ``send()`` is called.
 
 * ``attachments``: A list of attachments to put on the message. These can
-  be either :class:`~email.mime.base.MIMEBase` instances, or ``(filename,
-  content, mimetype)`` triples.
+  be either :class:`~email.mime.base.MIMEBase` instances, or a named tuple
+  with attributes ``(filename, content, mimetype)``.
+
+  .. versionchanged:: 5.2
+
+    ``attachments`` is now a list of named tuples as opposed to regular tuples.
```

Comment:
```suggestion
    In older versions, tuple items of ``attachments`` were regular tuples, 
    as opposed to named tuples.
```
Maybe 🤔 
We might also need to move this to the bottom of the `EmailMessage` definition